dist/images/ovnkube.sh

```diff
@@ -2404,10 +2391,7 @@ ovn-controller() {
   wait_for_event ovs_ready
 
   echo "=============== ovn-controller - (wait for ready_to_start_node)"
-  if [[ ${ovnkube_node_mode} != "dpu-host" ]] && [[ ${ovn_enable_interconnect} != "true" ]]; then
-    echo "=============== ovn-node - (wait for ready_to_start_node)"
-    wait_for_event ready_to_start_node
-  fi
```

Comment:
doesn't this totally revert the change from the first commit? Maybe it makes sense to squash the first 2 commits?

Reply:
indeed, will squash them